atomic/f23_spc/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN dnf -y install atomic && dnf clean all
 
 # TODO: Don't clone from scratch, instead add local working copy there
 RUN dnf -y install git python3 python3-devel python3-gobject-base \
-    python3-setuptools dbus python3-dbus && dnf clean all && \
+    python3-setuptools dbus python3-dbus python-requests && dnf clean all && \
```

Comment:
python-requests is not needed to build daemon. We have already tried to build f23 SPC using this dockerfile and the build was successful. I think it is only needed to build atomic. But in this Dockerfile we don't build atomic, we install atomic using dnf. Compare this with other Dockerfiles.